third_party/xla_client/xrt_computation_client.cc

```diff
@@ -55,9 +56,27 @@ XrtComputationClient::TransferToServer(
 
       string device = GetEffectiveDevice(literals[i].device);
       const string& xrt_device = TorchDeviceToXrtDevice(device);
-      xrt::XLAAllocation alloc;
-      *alloc.mutable_value() = literal.ToProto();
-      tensorflow::Input::Initializer feed_value(alloc.SerializeAsString());
+      tensorflow::Tensor tensor(
+          XlaTypeToDataType(literal.shape().element_type()),
+          MakeEquivalentTensorShape(literal.shape()));
+      auto tdata = tensor.tensor_data();
+      XLA_CHECK_EQ(tdata.size(), literal.size_bytes());
+      // This is an hack.
```

Comment:
`a hack`

Reply:
I think, since the 'h' is transparent, should be 'an'. but you can find both ways around:

https://www.merriam-webster.com/words-at-play/is-it-a-or-an

If you are picky I can change it (that comment will be going away soon anyway ;) )